c/Catch2/Catch2-2.13.8.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'CMakeMake'
+
+name = 'Catch2'
+version = '2.13.8'
+
+homepage = 'https://github.com/catchorg/Catch2'
+description = """A modern, C++-native, header-only,
+ test framework for unit-tests, TDD and BDD
+ - using C++11, C++14, C++17 and later
+   (or C++03 on the Catch1.x branch)
+"""
+
+toolchain = SYSTEM
```

Comment:
We've learned the hard way that using a system toolchain here is a bad idea, see https://github.com/easybuilders/easybuild-easyconfigs/pull/19270